include/flecs/private/api_internals.h

```diff
@@ -49,6 +49,31 @@ typedef struct ecs_table_diff_t {
 } ecs_table_diff_t;
 
 
+#ifdef FLECS_SAFETY_LOCKS
```

Comment:
Rename to something more descriptive like `FLECS_MUT_ALIAS_LOCKS` or something. "SAFETY" doesn't have the same meaning outside a rust context.